subsys/debug/coredump/CMakeLists.txt

```diff
@@ -24,8 +24,8 @@ zephyr_library_sources_ifdef(
   )
```

Comment:
Something weird with the commits, I see two commit that do the same thing, please collapse into one (unless there's some reason I'm missing here).

A few other things in commit message:
- add reference to Intel ADSP in commit title 
- "ACSI" -> "ASCII" 
- memory window slot [1] -> I'd just describe this is written to the telemetry slot. discussing the future steps is a bit difficult here as this just prompts people to ask why not why, what is the telemetry structure and so forth. but let's see what other things. I think it would be ok to just describe what this patch does (i.e. write the dump to the telemetry slot).

Reply:
Okey, I think that I've made what you asked for.